sparkle/src/components/MultiPageSheet.tsx

```diff
@@ -0,0 +1,202 @@
+import * as React from "react";
+
+import { Button, Icon } from "@sparkle/components";
+import {
+  Sheet,
+  SheetClose,
+  SheetContainer,
+  SheetContent,
+  SheetDescription,
+  SheetFooter,
+  SheetHeader,
+  SheetTitle,
+  SheetTrigger,
+} from "@sparkle/components/Sheet";
+import { ChevronLeftIcon, ChevronRightIcon } from "@sparkle/icons/app";
+
+interface MultiPageSheetPage {
+  id: string;
+  title: string;
+  description?: string;
+  icon?: React.ComponentType;
+  content: React.ReactNode;
+}
+
+interface MultiPageSheetProps {
+  pages: MultiPageSheetPage[];
+  currentPageId: string;
+  onPageChange: (pageId: string) => void;
+  size?: React.ComponentProps<typeof SheetContent>["size"];
+  side?: React.ComponentProps<typeof SheetContent>["side"];
+  trapFocusScope?: boolean;
+  showNavigation?: boolean;
+  footerContent?: React.ReactNode;
+  onSave?: () => void;
+  className?: string;
+}
+
+const MultiPageSheetRoot = Sheet;
+const MultiPageSheetTrigger = SheetTrigger;
+const MultiPageSheetClose = SheetClose;
+
+interface MultiPageSheetContentProps extends MultiPageSheetProps {
+  children?: never;
+}
+
+const MultiPageSheetContent = React.forwardRef<
+  React.ElementRef<typeof SheetContent>,
+  MultiPageSheetContentProps
+>(
+  (
+    {
+      pages,
+      currentPageId,
+      onPageChange,
+      size = "md",
+      side = "right",
+      trapFocusScope,
```

Comment:
Should this be false by default? I see a tooltip immediately when I open: 

<img width="617" alt="Screenshot 2025-05-23 at 16 34 28" src="https://github.com/user-attachments/assets/762f6fcd-f649-48ba-a257-568b3d462c2c" />

Reply:
I will probably have to tweak this depending on what we will put inside and where we will want the focus to be. I might have to rework the structure a little